DeratControl.Application/Perimeters/Commands/DeletePerimeterCommand.cs

```diff
@@ -0,0 +1,41 @@
+﻿using DeratControl.Application.Interfaces;
+using DeratControl.Application.Requests;
+using DeratControl.Application.Requests.Interfaces;
+using DeratControl.Domain.Entities;
+using DeratControl.Domain.Root;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DeratControl.Application.Perimeters.Commands
+{
+    public class RemovePerimeterCommand : IRequest
```

Comment:
maybe you should supply perimeter id? because after you create a perimeter you have its id?

Reply:
Okay I think your approach is good as well